mk/config.mk

```diff
@@ -226,3 +226,6 @@ ifneq ($(CFG_TA_GPROF_SUPPORT),y)
 $(error Cannot instrument user libraries if user mode profiling is disabled)
 endif
 endif
+
+# Enable Secure Data Path support in OP-TEE core
+CFG_WITH_SDP ?= n
```

Comment:
CFG_WITH_SDP ? CFG_SDP ? what's best ? 
i have no definitive opinion on that but that i don't really like too short shell  variables (potential conflicts with other settings/directives, related or not to OP-TEE).
Maybe switch to `CFG_SECURE_DATA_PATH=y/n` ? (sorry for the acronym fans :)